Constraints/DataLoader.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DataLoader.swift
+//  Constraints
+//
+//  Created by Angela Lee on 16/08/2022.
+//
+
+import Foundation
+
+public class DataLoader {
+    
+    func loadMessages() -> [MessagesData] {
+        
+        if let fileLocation = Bundle.main.url(forResource: "messages", withExtension: "json") {
```

Comment:
the file name should be as a parameter of the function

Reply:
Corrected